sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.service.ts

```diff
@@ -260,19 +266,22 @@ export class EducationProgramOfferingControllerService {
     if (creationErrors.length) {
       // If there is any failed result throw an error.
       const validationResults: OfferingBulkInsertValidationResultAPIOutDTO[] =
-        creationErrors.map((creationError) => {
-          const csvModel = csvModels[creationError.validatedOffering.index];
-          return {
-            recordIndex: creationError.validatedOffering.index,
-            locationCode: csvModel.institutionLocationCode,
-            sabcProgramCode: csvModel.sabcProgramCode,
-            startDate:
-              creationError.validatedOffering.offeringModel.studyStartDate,
-            endDate: creationError.validatedOffering.offeringModel.studyEndDate,
-            errors: [creationError.error],
-            warnings: [],
-          };
-        });
+        creationErrors
+          .map((creationError) => {
```

Comment:
I see here the formatting is different between existing and the modified code. why so ?

Reply:
Because of the addition of the `sort((a, b) => a.recordIndex - b.recordIndex)` at the end it is formatting is this way. Maybe to keep `.map` and `.sort` aligned.